deploy/6.6.StrategyFlamIncome.js

```diff
@@ -0,0 +1,50 @@
+module.exports = async ({ getNamedAccounts, deployments, getChainId }) => {
+    const { deploy, execute } = deployments;
+    let { USDT, WETH, deployer, unirouter } = await getNamedAccounts();
+    const chainId = await getChainId();
+    const controller = await deployments.get('StrategyControllerV2');
+    const vaultManager = await deployments.get('yAxisMetaVaultManager');
+    const Converter = await deployments.get('StableSwap3PoolConverter');
+
+    let flamIncomeUSDT;
```

Comment:
Add the project's mainnet addresses to `hardhat.config.js` and use them in `getNamedAccounts`.